packages/frontend/hooks/popover.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect } from 'react';
+
+export const useOutsideClick = (elementRef, handler, attached = true) => {
```

Comment:
What is it? Why is it here?

Reply:
Maybe it wasn't deleted because of merging